app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -0,0 +1,40 @@
+.header-sidebar-toggle {
+  margin-right: 1em;
+  margin-left: -10px;
```

Comment:
Any way to avoid this?

Reply:
What is the reason we want to avoid a left margin?

Reply:
It's not the left margin per se, it's the negative margin. I see why you need this though, to line up the icon correctly, because the button has padding. We can live with it.

Reply:
Actually I would have to defer to @awesomerobot on this caused I only worked on the component parts.